schema/sql/api.sql

```diff
@@ -1,4 +1,6 @@
 -- -*- mode: sql; sql-product: postgres -*-
+create schema pgcrypto;
+create extension pgcrypto schema pgcrypto;
```

Comment:
`create extension` is one of the very few things you should use `if not exists` with.

Reply:
TIL

Reply:
This is because many postgres instances come with extensions pre-loaded.
Some extensions also have cluster-wide pieces, which means that other databases in the same cluster might have already loaded the extension.